pkg/rbac/sharedData.go

```diff
@@ -198,8 +198,9 @@ func (shared *SharedData) GetManagedClusters(cache *Cache, ctx context.Context)
 	}
 
 	for _, item := range resourceObj.Items {
-		managedClusters = append(managedClusters, item.GetName())
-
+		if item.GetName() != "local-cluster" {
```

Comment:
This works for now, but I wonder if there's any way to avoid the dependency on the "local-cluster" name.

FYI: There's an RFE open to allow users to rename it.  I've heard this requirement several times before and it came up at today's architecture call again.

Reply:
That's a good point. Will merge for now and add improvement to the umbrella story.